lectures/aiyagari_jax.md

```diff
@@ -0,0 +1,925 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"colab_type": "text", "id": "view-in-github"}
```

Comment:
@shlff please remove all this meta data inserted by colab --- it should  not be in our md files

Reply:
Thanks for your nice comments @jstac . 

The commit below fixes these issues and updates the correct ``math-label`` environment for math equations.